src/test/java/LottoShopTest.java

```diff
@@ -0,0 +1,55 @@
+import domain.Lotto;
+import domain.LottoShop;
+import domain.Lottos;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LottoShopTest {
+    @SuppressWarnings("NonAsciiCharacters")
+
+    @DisplayName("자동_로또_수를_계산한다")
+    @Test
+    void 자동_로또_수를_계산한다() {
+        // given
+        LottoShop lottoShop = new LottoShop();
+        int inputMoney = 5000;
+        int manualCount = 2;
```

Comment:
```suggestion
        var lottoShop = new LottoShop();
        var inputMoney = 5000;
        var manualCount = 2;
```
P3) 이런 방식도 있다는걸 참고하면 좋아보입니다
이거는 제 테스트 코드 스타일이긴한데, 제 팀프로젝트 테스트 코드는 main 패키지 코드를 아는 사람들만 보기 때문에 타입 추론을 사용했었어요.
어차피 `new LottoShop()`, `5000`, `2`는 모두가 타입을 아는데, 굳이 `LottoShop`, `int`, `int`를 추가해서 저걸 읽게 할 필요가 있을까하는 느낌?
저희가 하루에 코드로 읽는 단어의 총량이 정해져있다면 저것만 수정해도 단어 3개를 안보게 되고, 다른 곳에서 단어 3개를 볼 수 있으니까요.
이건 타입추론 자체가 호불호가 크기 때문에 참고만...